matsim/src/main/java/org/matsim/core/population/PersonAttributes.java

```diff
@@ -0,0 +1,41 @@
+package org.matsim.core.population;
```

Comment:
I am not sure about that. If any call on it results in an exception anyways, I would rather directly remove the ObjectAttributes getter from the Population, or at least have it throw an exception, and add a package-private getter for PopulationUtils. I will try a proposal in that direction.

Reply:
I think the main reason for not doing that is that it throws the exception at the wrong place.  There is a lot of code that first calls `population.getPersonAttributes()`, then passes the result around, and only much later calls `personAttributes.put/get/...`.  But it is the latter place where one needs to replace it with the static method.  But I have no strong opinion here.